Library/Homebrew/extend/ARGV.rb

```diff
@@ -64,6 +64,10 @@ def build_from_source?
     switch?("s") || include?("--build-from-source")
   end
 
+  def silent?
+    include?("--download-silently")
+  end
```

Comment:
I would suggest that you name this method the same as the flag, following the pattern that the other flags follow.

```suggestion
  def download-silently?
    include?("--download-silently")
  end
```